src/Api/Endpoints/Magic.cs

```diff
@@ -38,22 +39,31 @@ public static void MapMagicEndpoints(this IEndpointRouteBuilder app)
             .RequireSecretKey()
             .WithTags(OpenApiTags.MagicLinks);
 
-        group.MapPost("/send", async (
-                SendMagicLinkRequest request,
-                IFeatureContextProvider provider,
-                MagicLinkService magicLinkService
-            ) =>
-            {
-                if (!(await provider.UseContext()).IsMagicLinksEnabled)
-                    throw new ApiException("You must enable Magic Links in settings in order to use this feature.", 403);
-
-                await magicLinkService.SendMagicLinkAsync(request.ToDto());
-
-                return NoContent();
-            })
+        group.MapPost("/send", SendMagicLinkAsync)
             .WithParameterValidation()
-            .RequireSecretKey()
-            .RequireCors("default")
             .RequireRateLimiting(RateLimiterPolicy);
     }
+
+    /// <summary>
+    /// Sends email containing a magic link template allowing users to login.
+    ///
+    /// Warning: Verify the e-mail address matches the user identifier in your backend.
```

Comment:
Nit: maybe slightly better wording?

```suggestion
    /// Warning: it's up to you to verify that the provided e-mail address belongs to the specified user.
```

Reply:
No they should validate it